migrator/zenodo_rdm_migrator/actions/drafts.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2023 CERN.
+#
+# Invenio-RDM-Migrator is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Invenio RDM migration drafts row load module."""
+
+
+from invenio_rdm_migrator.actions import TransformAction
+from invenio_rdm_migrator.load.postgresql.transactions.operations import OperationType
+from invenio_rdm_migrator.streams.actions import RDMDraftCreateAction
+from invenio_rdm_migrator.transform import IdentityTransform
+
+from ..transform.records import ZenodoRecordTransform
+
+
+class ZenodoDraftCreateAction(TransformAction):
+    """Zenodo to RDM draft creation action."""
+
+    name = "create-zenodo-draft"
+    load_cls = RDMDraftCreateAction
+
+    @classmethod
+    def matches_action(cls, tx):  # pragma: no cover
+        """Checks if the data corresponds with that required by the action."""
+        rules = {
+            "pidstore_pid": OperationType.INSERT,
+            "files_bucket": OperationType.INSERT,
+            "records_metadata": OperationType.INSERT,
+        }
+
+        for operation in tx.operations:
+            table_name = operation["source"]["table"]
+
+            rule = rules.pop(table_name, None)
+            # rule can be None, e.g. draft creation has two rows on pidstore_pid
```

Comment:
@slint this is bad :( we need to find a better way for those "ignored ones", the use case is when there are multiple for one table and we have to ignore some of them. Could lead to errors.